avro-data/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -672,6 +679,8 @@ && crossReferenceSchemaNames(schema, typeSchema, scrubInvalidNames)) {
             return typeSchema;
           }
         }
+      } else if (avroSchema.getType() == org.apache.avro.Schema.Type.MAP) {
+        // do nothing, this is from Protobuf Map
```

Comment:
Is this still needed?  If so maybe change the comment to // do nothing, this is an optional map, such as in Protobuf

Reply:
You're right, this is not needed anymore. Thanks for catching it.